app/templates/domains.html

```diff
@@ -1,6 +1,11 @@
 <div class="column">
 
     <div class="ui cards" id="domain_cards">
+        {% if errorMessage %}
+        <div>
+                {{errorMessage}}
```

Comment:
It is functional, but you could enhance it with a message from the semantic framework. Here is a link to the correct component: [Error Messages](https://semantic-ui.com/collections/message.html#negative--error)